src/features/wasm-bigint.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'WebAssembly Integration with JavaScript BigInt'
+author: 'Alon Zakai'
+avatars:
+  - 'alon-zakai'
+date: 2020-11-12
+tags:
+  - WebAssembly
+  - ECMAScript
+description: 'BigInts make it easy to pass 64-bit integers between JavaScript and WebAssembly. This post explains what that means and why it’s useful, which includes making things simpler for developers, letting code run more quickly, and also speeding up build times.'
+tweet: 'TODO'
+---
+The
+[JS-BigInt-Integration](https://github.com/WebAssembly/JS-BigInt-integration) feature makes it easy to pass 64-bit integers between JavaScript and WebAssembly. This post explains what that means and why it’s useful, which includes making things simpler for developers, letting code run more quickly, and also speeding up build times.
```

Comment:
Newlines are preserved, so this renders as a line break (can see at https://v8-dev--pr502-bigint-5waus1ta.web.app/features/wasm-bigint).

```suggestion
The [JS-BigInt-Integration](https://github.com/WebAssembly/JS-BigInt-integration) feature makes it easy to pass 64-bit integers between JavaScript and WebAssembly. This post explains what that means and why it’s useful, which includes making things simpler for developers, letting code run more quickly, and also speeding up build times.
```